mergin/merginproject.py

```diff
@@ -347,15 +347,19 @@ def get_push_changes(self):
                 self.geodiff.create_changeset(origin_file, current_file, diff_file)
                 if self.geodiff.has_changes(diff_file):
                     diff_size = os.path.getsize(diff_file)
-                    file["checksum"] = file["origin_checksum"]  # need to match basefile on server
-                    file["chunks"] = [str(uuid.uuid4()) for i in range(math.ceil(diff_size / UPLOAD_CHUNK_SIZE))]
-                    file["mtime"] = datetime.fromtimestamp(os.path.getmtime(current_file), tzlocal())
-                    file["diff"] = {
-                        "path": diff_name,
-                        "checksum": generate_checksum(diff_file),
-                        "size": diff_size,
-                        "mtime": datetime.fromtimestamp(os.path.getmtime(diff_file), tzlocal()),
-                    }
+                    if diff_size > 0:
```

Comment:
"if diff_size > 0" is not necessary, because the self.geodiff.has_changes(diff_file) check above is equivalent of that